src/components/MyKiva/GoalCard.vue

```diff
@@ -145,18 +156,30 @@ const ctaHref = computed(() => {
 const achievementGoalImg = computed(() => {
 	const contentfulCategory = getContentfulKey(props.userGoal?.category) || '';
 	if (!contentfulCategory) return '';
-	const key = `my-kiva-${contentfulCategory}-journey`;
+	const key = `my-kiva-${'women'}-journey`;
```

Comment:
Does the logic here need to change? Should this be `contentfulCategory`? What happens for "all loans" goal?

Reply:
What happens for the "all loans" category? Do we need a new contentful entry?